cmd/bucket-replication.go

```diff
@@ -838,9 +843,8 @@ func NewReplicationPool(ctx context.Context, o ObjectLayer, sz int) *Replication
 		ctx:                ctx,
 		objLayer:           o,
 	}
-	pool.Resize(sz)
-	// add long running worker for handling most recent failures/pending replications
-	go pool.AddMRFWorker()
+	pool.ResizeWorkers(opts.Workers)
+	pool.ResizeFailedWorkers(opts.FailedWorkers)
```

Comment:
should we increase the replicaCh size as well to 10K

Reply:
done